packages/common/src/broker-types/automod.ts

```diff
@@ -0,0 +1,27 @@
+import type { ValueResolvable } from '@sapphire/bitfield';
+import { BitField } from '@sapphire/bitfield';
+
+export enum Runners {
+	noop,
+}
+
+export type BaseRunnerResult<R extends Runners, T> = {
+	data: T;
+	runner: R;
+};
+
+export type NoopRunnerResult = BaseRunnerResult<Runners.noop, null>;
+
+export type RunnerResult = NoopRunnerResult;
+
+export const FilterIgnoresTypes = {
```

Comment:
Pretty sure this can just be an actual enum i.e.
```ts
enum FilterIgnoreTypes {
  urls = 1n << 0n,
  // etc
}
```

Reply:
I don't think that's actually possible atm, sadly.
```ts
export enum FilterIgnoresTypes {
	urls = 1n << 0n, // Type 'bigint' is not assignable to type 'number' as required for computed enum member values. ts(18033)
        // ...
```

Reply:
TIL